blt/blt.yml

```diff
@@ -72,5 +75,7 @@ command-hooks:
   post-deploy:
     dir: '${docroot}'
     command: null
+simplesamlphp: true
```

Comment:
I feel like this should go into separate blt.yml file for the environments.  You can do <ENV>.blt.yml and override settings on a per environment basis. 

ODEs configs are impossible, but as we discussed previously, ODEs will not get SSO.
One final note to remember, `ci` is the pipelines environment.

Reply:
This is a flag which controls whether the `simplesamlphp` library is copied into the deploy artifact, it does not enable any modules. Leaving in here avoids some complexity by keeping all of the BLT configuration for SAML in once place.

Reply:
This is a flag which controls whether the `simpleSAMLphp` configuration files are copied into the expected location.  It does not directly toggle the authentication module on/off.

Moving this out would mean BLT configuration for the authorization module needs to be done in two places. Perhaps *all* configuration for a given environment (e.g. the module enable/uninstall list) should be moved out?